packages/plugin-link/src/Link.ts

```diff
@@ -65,8 +65,8 @@ export class Link<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
         components: [
             {
                 id: 'link',
-                async render(): Promise<OwlNode[]> {
-                    return [new OwlNode({ Component: LinkComponent, props: {} })];
+                async render(editor: JWEditor, params?: Record<string, any>): Promise<OwlNode[]> {
```

Comment:
@Zinston @Goaman Shall we name the component params as `props` or do you think it might get confusing if people use them as something else than props in their component's `render` method ?

Reply:
Why would you want to use them as something else than props?
For me, props is good because it's the standard way of naming the parameters that a component receive.

Reply:
cc @sebgeelen for renaming the `params` parameter into `props` up along the entire function call chain.